### PR TITLE
Add feedback store and form

### PR DESCRIPTION
**Pull request by ProbablyFaiz**

### CODE REVIEW

**Comment by ProbablyFaiz** [comment]
[React Issue 2: Add FeedbackStore, and add FeedbackForm](https://trello.com/c/D5KjhtXm/2-react-issue-2-add-feedbackstore-and-add-feedbackform)

File: app/javascript/image_sharing/components/Feedback.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { observer } from 'mobx-react';
+import { Form, FormGroup, Label, Input, Button } from 'reactstrap';
+import FeedbackStore from '../stores/FeedbackStore';
+
+@observer
```
Comment (bhargav265) [inline app/javascript/image_sharing/components/Feedback.js:7]: can you inject the store directly here?

File: app/javascript/image_sharing/components/App.js [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
-  /* Add Prop Types check*/
+  static propTypes = {
+    stores: PropTypes.object.isRequired
+  };
+
   render() {
     return (
       <div>
         <Header title="Send Feedback" />
+        <Feedback feedbackStore={this.props.stores.feedbackStore} />
```
Comment (bhargav265) [inline app/javascript/image_sharing/components/App.js:16]: can you add a test for this in App.spec.js?

File: app/javascript/image_sharing/stores/FeedbackStore.js [hunk 3]
```diff
@@ -1,5 +1,18 @@
...
+
+  @action
+  setFeedbackValue = (feedback) => {
+    this.feedbackValue = feedback;
+  };
+
+  @action
+  setNameValue = (name) => {
+    this.nameValue = name;
+  }
```
Comment (bhargav265) [inline app/javascript/image_sharing/stores/FeedbackStore.js:15]: Can just use 1 function to set the attribute. You may store the attributes in a hash

File: app/javascript/image_sharing/test/FeedbackStore.spec.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+import FeedbackStore from '../stores/FeedbackStore';
+
+describe('FeedbackStore', () => {
+  it('has actions that set observable values', () => {
```
Comment (bhargav265) [inline app/javascript/image_sharing/test/FeedbackStore.spec.js:6]: can you test the initial values as well?

Comment (ProbablyFaiz) [reply app/javascript/image_sharing/test/FeedbackStore.spec.js:6]: Done.

**Pull request merged**